packages/components/src/internal/components/entities/GridAliquotViewSelector.tsx

```diff
@@ -28,12 +28,7 @@ interface Props {
     actions?: Actions;
     initAliquotMode?: ALIQUOT_FILTER_MODE; // allow to set aliquot filter from a init value
     queryModel?: QueryModel;
-    updateFilter?: (
-        filter: Filter.IFilter,
-        filterColumnToRemove?: string,
-        newModel?: ALIQUOT_FILTER_MODE,
-        queryModel?: QueryModel
-    ) => void;
+    updateFilter?: (filter: Filter.IFilter, newModel: ALIQUOT_FILTER_MODE) => void;
```

Comment:
nit: The associated `GridAliquotViewSelector.spec.tsx` should be converted to RTL with these changes.